test/test.ml

```diff
@@ -19,4 +19,14 @@ let adder = Function.add_function wasm_mod "adder" params results [||] add
 
 let _ = Module.print wasm_mod
 
+let byts, _ = Module.write wasm_mod None
```

Comment:
@ospencer I added tests for calling `Module.write` and `Module.read` to test the binary conversions in JSOO

Reply:
This looks good!